src/components/Navigation.vue.orig

```diff
@@ -0,0 +1,209 @@
+			</ActionButton>
+		</Actions>
+		<h2 class="photos-navigation__title">
+			{{ name }}
+		</h2>
+		<Actions v-if="!isRoot && showActions" class="photos-navigation__share">
+			<ActionButton
+				icon="icon-shared"
+				@click="showSidebar">
+				{{ t('photos', 'Share this folder') }}
+			</ActionButton>
+		</Actions>
+	</div>
+</template>
+
+<script>
+import Actions from '@nextcloud/vue/dist/Components/Actions'
+import ActionButton from '@nextcloud/vue/dist/Components/ActionButton'
+import isMobile from '@nextcloud/vue/dist/Mixins/isMobile'
+
+export default {
+	name: 'Navigation',
+
+	components: {
+		ActionButton,
+		Actions,
+	},
+
+	mixins: [isMobile],
+	inheritAttrs: false,
+
+	props: {
+		basename: {
+			type: String,
+			required: true,
+		},
+		filename: {
+			type: String,
+			required: true,
+		},
+		showActions: {
+			type: Boolean,
+			default: false,
+		},
+		rootTitle: {
+			type: String,
+			default: t('photos', 'Photos'),
+		},
+	},
+
+	computed: {
+		isRoot() {
+			return this.filename === '/'
+		},
+		name() {
+			if (this.isRoot) {
+				return this.rootTitle
+			}
+			return this.basename
+		},
+		parentPath() {
+			const path = this.filename.split('/')
+			path.pop()
+			const parent = path.join('/')
+			return this.isRoot || parent.trim() === ''
+				? '/'
+				: path.join('/')
+		},
+		parentName() {
+			return this.parentPath && this.parentPath.split('/').pop()
+		},
+		backToText() {
+			if (this.parentPath === '/') {
+				return t('photos', 'Back to {folder}', { folder: this.rootTitle })
+			}
+			return t('photos', 'Back to {folder}', { folder: this.parentName })
+		},
+
+		/**
+		 * We do not want encoded slashes when browsing by folder
+		 * so we generate a new valid route object, get the final url back
+		 * decode it and use it as a direct string, which vue-router
+		 * does not encode afterwards
+		 * @returns {string|object}
+		 */
+		to() {
+			// always remove first slash, the router
+			// manage it automatically
+			const regex = /^\/?(.*)/i
+			const path = regex.exec(this.parentPath)[1]
+
+			// apply to current route
+			const { name, params } = Object.assign({}, this.$route, {
+				params: { path },
+			})
+
+			// return the full object as we don't care about
+			// an empty path if this is route
+			if (path === '') {
+				return { name }
+			}
+
+			// returning a string prevent vue-router to encode it again
+			return decodeURIComponent(this.$router.resolve({ name, params }).resolved.path)
+		},
+	},
+
+	methods: {
+		folderUp() {
+			this.$router.push(this.to)
+		},
+		showSidebar() {
+			OCA.Files.Sidebar.open(this.filename)
+
+		},
+	},
+}
+</script>
+
+<style lang="scss" scoped>
+.icon-confirm {
+	transform: rotate(180deg)
+}
+
+.photos-navigation {
+	display: flex;
+	position: absolute;
+	height: 44px;
+	align-items: center;
+	&__title {
+		margin: 0;
+	}
+<<<<<<< HEAD
+	&__share {
+		margin-left: 10px;
+=======
+	&--mobile {
+		margin-left: 50px;
+>>>>>>> Shift navigation on mobile view
```

Comment:
Some merge conflict here?

Reply:
We can remove this file, wrongly commited